lib/system.nim

```diff
@@ -529,10 +529,7 @@ proc `..`*[T](b: sink T): HSlice[int, T]
   ##   echo a[.. 2] # @[10, 20, 30]
   result = HSlice[int, T](a: 0, b: b)
 
-when defined(hotCodeReloading):
-  {.pragma: hcrInline, inline.}
-else:
-  {.pragma: hcrInline.}
+{.pragma: hcrInline.}
```

Comment:
Could we get rid of this too?

Reply:
Sure. I only kept it in order to reduce the size of the change-set.

Reply:
Makes sense, but more feels is more points. 😁🥳